mapper/dm_agent/src/batcher.rs

```diff
@@ -274,17 +263,19 @@ mod tests {
         Ok(())
     }
 
+    /*
```

Comment:
Why this test has been commented out?

Either remove it, fix it or add the appropriate `#[cfg(feature = "...")]`.